Kernel/Graphics/VirtIOGPU/FramebufferDevice.cpp

```diff
@@ -45,15 +45,15 @@ KResult FrameBufferDevice::create_framebuffer()
     }
     m_framebuffer_sink_vmobject = TRY(Memory::AnonymousVMObject::try_create_with_physical_pages(pages.span()));
 
-    MutexLocker locker(m_gpu.operation_lock());
+    MutexLocker locker(static_cast<GraphicsAdapter&>(*m_graphics_adapter).operation_lock());
```

Comment:
Maybe wrap the `static_cast<GraphicsAdapter&>(*m_graphics_adapter)` within a private method?

Reply:
I did that in a separate commit. I might squash it to the commit that created that pattern if it seems right to you.